### PR TITLE
Add CC license header to documentation

### PR DESCRIPTION
**Pull request by SimonBiggs**
Added the following at the top of each of the regulatory documents:

```md
<!--
Copyright (C) 2022 Radiotherapy AI Holdings Pty Ltd
Copyright (C) 2021-2022 OpenRegulatory (OpenReg GmbH)
This work is licensed under the Creative Commons Attribution 4.0 International
License. <http://creativecommons.org/licenses/by/4.0/>.

Original work by OpenRegulatory available at
<https://github.com/openregulatory/templates>
-->
```

As a result of opening the templates, changing something, and saving, my markdown autoformatter was also run. As such there will be a range of autoformatting changes throughout the templates also.

---

No manual changes (except for adding of the license header to each file) was undergone within this pull request.

### CODE REVIEW

File: docs/1-qms/13485-mapping.md [hunk 1]
```diff
@@ -9,39 +19,39 @@ This table maps all requirements of the ISO 13485:2016 (by section) to the relev
...
-| 8.2.6   | Surveillance and Measurement of Products          | *SOP Post-Market Surveillance*                                                                                   |
-| 8.3     | Control of Nonconforming Products                 | *SOP Corrective and Preventive Actions*<br>*SOP Incident Reporting*                                              |
-| 8.4     | 8.4 Data Analysis                                 | *SOP Management Review*                                                                                          |
-| 8.5     | 8.5 Improvement: Corrective and Preventive Action | *SOP Corrective and Preventive Actions*                                                                          |
+| ------- | ------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| 4.1     | General QMS Requirements                          | _Quality Management Manual_<br>_SOP Management Review_<br>_SOP Purchasing_<br>_SOP Software Validation_          |
```
Comment (sjswerdloff) [inline docs/1-qms/13485-mapping.md:23]: Making even small unique changes like this in a Single Change Multiple Files makes reviewing more challenging.  A quick overview is possible with SCMF, but once other changes show up, each file has to be inspected carefully...

File: docs/2-techdoc/62304/checklist-software-release.md [hunk 2]
```diff
@@ -24,22 +34,22 @@ The following documents are up to date:
...
+| Item                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | Yes | No  | Comment |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- | --- | ------- |
```
Comment (sjswerdloff) [inline docs/2-techdoc/62304/checklist-software-release.md:47]: this looks weird, and doesn't line up with the Item/yes/no/comment particularly well, even allowing for the white space showing after Item.  If it renders correctly, then no issue

File: docs/2-techdoc/62304/software-architecture-description.md [hunk 3]
```diff
@@ -6,10 +16,20 @@ IEC 62304, para. 5.3.1 and 5.3.2 [class B, C]
...
-* SOP Software Development
-* User needs / stakeholder requirements
-* Design input / software requirements
-* (...)
+- SOP Software Development
+- User needs / stakeholder requirements
+- Design input / software requirements
+- (...)
+
+<!--
```
Comment (sjswerdloff) [inline docs/2-techdoc/62304/software-architecture-description.md:24]: this is duplicate, and probably needs to be deleted

File: docs/2-techdoc/62304/software-architecture-description.md [hunk 4]
```diff
@@ -25,47 +45,77 @@ components and software units.
...
-> * Runtime (e.g. JVM)
-> * Deployment (e.g. Docker container)
-> * User group
+> - Function: taking medical images as input and output a prediction.
+> - Software safety classification and rationale
+> - Runtime (e.g. JVM)
+> - Deployment (e.g. Docker container)
+> - User group
+
+<!--
```
Comment (sjswerdloff) [inline docs/2-techdoc/62304/software-architecture-description.md:74]: this is duplicate and probably needs to be deleted also

File: docs/2-techdoc/62304/software-architecture-description.md [hunk 5]
```diff
@@ -25,47 +45,77 @@ components and software units.
...
 > subdivided into other items”. For example:
 >
-> * Wearable device poller (regularly checks whether wearable device has new data and downloads it)
-> * Notification service (sends messages to Apple / Google for push notifications of mobile apps)
-> * (...)
+> - Wearable device poller (regularly checks whether wearable device has new data and downloads it)
+> - Notification service (sends messages to Apple / Google for push notifications of mobile apps)
+> - (...)
+
+<!--
```
Comment (sjswerdloff) [inline docs/2-techdoc/62304/software-architecture-description.md:94]: dupe, needs to be deleted

File: docs/2-techdoc/62304/software-architecture-description.md [hunk 6]
```diff
@@ -25,47 +45,77 @@ components and software units.
...
+-->
 
 # 7. Database
 
 > Describe your databases. For example:
 >
-> * Relational database: Postgres v14
+> - Relational database: Postgres v14
+
+<!--
```
Comment (sjswerdloff) [inline docs/2-techdoc/62304/software-architecture-description.md:110]: dupe... delete

File: docs/3-general/instructions-for-use.md [hunk 7]
```diff
@@ -1,21 +1,30 @@
...
+This work is licensed under the Creative Commons Attribution 4.0 International
+License. <http://creativecommons.org/licenses/by/4.0/>.
+
+Original work by OpenRegulatory available at
+<https://github.com/openregulatory/templates>
+-->
 
 # User Manual [enter Product Name]
 
+| ISO 13485:2016 Section | Document Section |
```
Comment (sjswerdloff) [inline docs/3-general/instructions-for-use.md:13]: This changes the location of this ISO reference with respect to the 
# User Manual
entry. 
Is that intentional?  If not intentional, is it acceptable?

**Review by sjswerdloff — CHANGES_REQUESTED** [review]
the duplicate entries in one of the files need to be removed.
If those are intentional, that seems a bit strange, and some explanation of why they are needed would be useful.
please also at least comment on rearrangement of ISO reference with respect to User Document line (in that particular file).

**Comment by sjswerdloff** [comment]
It's also a bit disconcerting to see a lot of "white space" and "dashes for lining things up" types of changes in general, I would have hoped that markdown would make column alignment more programmatic (although I imagine that's a messy thing given how much work goes in to HTML for that purpose).

File: docs/1-qms/13485-mapping.md (hunk 1 above)
Comment (SimonBiggs) [reply docs/1-qms/13485-mapping.md:23]: Yeah, it was a bit annoying that my auto-formatter ran. In the future, I'll auto-format everything first, separately, and then make the other bulk change in a separate PR.

File: docs/2-techdoc/62304/checklist-software-release.md (hunk 2 above)
Comment (SimonBiggs) [reply docs/2-techdoc/62304/checklist-software-release.md:47]: I suspect these are due to word wrap on your screen. If word wrap is turned off they should all line up after the auto-formatter has run...

File: docs/2-techdoc/62304/software-architecture-description.md (hunk 3 above)
Comment (SimonBiggs) [reply docs/2-techdoc/62304/software-architecture-description.md:24]: Yes, sorry about that. I should have found them before submitting for review. Thanks @sjswerdloff.

File: docs/3-general/instructions-for-use.md (hunk 7 above)
Comment (SimonBiggs) [reply docs/3-general/instructions-for-use.md:13]: User Manual itself is a title and not a part of the table. I would be quite surprised if there are any output changes as a result of the markdown auto-formatter...

File: docs/2-techdoc/62304/software-architecture-description.md (hunk 3 above)
Comment (SimonBiggs) [reply docs/2-techdoc/62304/software-architecture-description.md]: Fixed in b91dc23

File: docs/2-techdoc/62304/checklist-software-release.md (hunk 2 above)
Comment (SimonBiggs) [reply docs/2-techdoc/62304/checklist-software-release.md:47]: When viewing locally, with word-wrap turned off it looks like the following:

image

image

I'll be pulling tables out into csv files when I step through the templates (which will fix this up)

File: docs/1-qms/13485-mapping.md (hunk 1 above)
Comment (SimonBiggs) [reply docs/1-qms/13485-mapping.md:23]: And just to confirm, this was the autoformatter, not a manual change.

**Comment by SimonBiggs** [comment]
> It's also a bit disconcerting to see a lot of "white space" and "dashes for lining things up" types of changes in general, I would have hoped that markdown would make column alignment more programmatic (although I imagine that's a messy thing given how much work goes in to HTML for that purpose).

Actually, markdown doesn't need the whitespace. The autoformatter makes those changes just so that the tables are aligned and easier to read in plain text. The alignment of course needs word-wrap turned off... which makes this all quite painful for very long lines.

...when going through the documents I will try to remove tables where they are not helpful, but I will also utilise csv imports where I can to pull the tables out into their own files. See where I have included csv in other docs already:

https://github.com/RadiotherapyAI/RadiotherapyAI/blob/987975e96af309f43725f9efa66d8a7cf609fd01/docs/2-techdoc/14971/risk-table-fmea/index.md?plain=1#L126-L149

**Comment by SimonBiggs** [comment]
> Making even small unique changes like this in a Single Change Multiple Files makes reviewing more challenging.

Let me know, I'd be happy to close this PR altogether, and start again, splitting it into two. Let me know.

**Comment by sjswerdloff** [comment]
> > Making even small unique changes like this in a Single Change Multiple Files makes reviewing more challenging.
> 
> Let me know, I'd be happy to close this PR altogether, and start again, splitting it into two. Let me know.

First time I gripe but move forward.
If there's a next time... I will request new PRs splitting the changes.

**Review by sjswerdloff — APPROVED** [review]

**Comment by SimonBiggs** [comment]
> First time I gripe but move forward.
> If there's a next time... I will request new PRs splitting the changes.

Thanks @sjswerdloff, I certainly want to optimise in favour of your time. If PR format is painful I'm more than happy to pull it down and try again. Definitely want to optimise in favour of ease of review, and happy to have PRs take more time in order to achieve that.

Thanks for this one 😊

**Pull request merged**